swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition=0):
```

Comment:
We want the category of any `Clothing` instance to be set to `"Clothing"`. In that case, do we need to have a parameter in the constructor for `category`? What would happen if this class was called like
```python
my_clothing = Clothing(category="definitely not clothing!", condition=5)
```